doc/man1/ciphers.pod

```diff
@@ -715,34 +715,34 @@ listed here because some ciphers were excluded at compile time.
 
 Verbose listing of all OpenSSL ciphers including NULL ciphers:
 
- openssl ciphers -v 'ALL:eNULL'
+ openssl ciphers -v "ALL:eNULL"
 
 Include all ciphers except NULL and anonymous DH then sort by
 strength:
 
- openssl ciphers -v 'ALL:!ADH:@STRENGTH'
+ openssl ciphers -v "ALL:!ADH:@STRENGTH"
```

Comment:
Unfortunately, '!' has special meaning to the shell when within double quotes. (multiple occurrences)